modules/jumpstart_ui/src/Plugin/paragraphs/Behavior/HeroPatternBehavior.php

```diff
@@ -88,8 +90,34 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
       '#description' => $this->t('Position the text over the image on larger screens'),
       '#default_value' => $paragraph->getBehaviorSetting('hero_pattern', 'overlay_position', 'left'),
       '#options' => [
-        'left' => $this->t('Left'),
-        'right' => $this->t('Right'),
+        'left' => $this->t('Left Card'),
+        'right' => $this->t('Right Card'),
+        'center' => $this->t('Center Transparent Overlay'),
+      ],
+    ];
+    $form['overlay_color'] = [
+      '#type' => 'select',
```

Comment:
Should this be hidden if the position is left or right card? if so here's some documentation to produce that show/hide behavior: https://www.drupal.org/docs/drupal-apis/form-api/conditional-form-fields